applications/main/bad_usb/bad_usb_script.c

```diff
@@ -112,18 +94,21 @@ static const char ducky_cmd_comment[] = {"REM"};
 static const char ducky_cmd_id[] = {"ID"};
 static const char ducky_cmd_delay[] = {"DELAY "};
 static const char ducky_cmd_string[] = {"STRING "};
+static const char ducky_cmd_stringln[] = {"STRINGLN "};
 static const char ducky_cmd_defdelay_1[] = {"DEFAULT_DELAY "};
 static const char ducky_cmd_defdelay_2[] = {"DEFAULTDELAY "};
 static const char ducky_cmd_stringdelay_1[] = {"STRINGDELAY "};
 static const char ducky_cmd_stringdelay_2[] = {"STRING_DELAY "};
 static const char ducky_cmd_repeat[] = {"REPEAT "};
 static const char ducky_cmd_sysrq[] = {"SYSRQ "};
+static const char ducky_cmd_hold[] = {"HOLD "};
+static const char ducky_cmd_release[] = {"RELEASE "};
 
 static const char ducky_cmd_altchar[] = {"ALTCHAR "};
 static const char ducky_cmd_altstr_1[] = {"ALTSTRING "};
 static const char ducky_cmd_altstr_2[] = {"ALTCODE "};
 
-static const uint8_t numpad_keys[10] = {
+/*static const uint8_t numpad_keys[10] = {
```

Comment:
We don't leave dead code. Please remove everything that is not used anymore.

Reply:
No problem, it has been fixed and I also added the issue number in the description as recommended, sorry it took a while I was offline for a few days.